src/Microsoft.ML.AutoML/Tuner/Flow2.cs

```diff
@@ -56,6 +54,11 @@ public Flow2(SearchSpace.SearchSpace searchSpace, Parameter initValue = null, bo
             {
                 _step = _stepUpperBound;
             }
+
+            if (rng != null)
```

Comment:
>(rng != null) [](http://example.com/codeflow?start=15&length=13)

nit: you don't need this condition I guess.

Reply:
### Resolved